src/constants/octokit.ts

```
@@ -0,0 +1,5 @@
+import { Octokit } from '@octokit/core'
+
+export const octokit = new Octokit({
+  auth: process.env.REACT_APP_GITHUB_ACCESS_TOKEN,
```

We may want to throw an error on load if the .env var isn't set